### DISCUSSION TITLE
How do I make the cursor wrap back to top?

### DISCUSSION THREAD

**Discussion by UtkarshVerma** [sxyazi/yazi#672] [1. Q&A] [answered]
When I go to the last file, I want the cursor to jump to top when I press down again. Is this possible?

**Comment by UtkarshVerma** [comment]
Could someone please shed some light on this, or maybe point me towards something?

**Comment by sxyazi** [comment, marked as answer]
I added a tip for this, see https://yazi-rs.github.io/docs/tips#file-navigation-wraparound

**Reply by UtkarshVerma** [reply]
Thanks a lot. Works like a charm.

**Reply by sxyazi** [reply]
I am considering making it the default behavior, but it would be a breaking change for existing users (user configurations include `arrow 99999` and `arrow -99999` as bottom and top), I need to think about it some more...

**Reply by UtkarshVerma** [reply]
Yeah, it actually makes sense for the cursor to wrap by default. Or maybe you could expose a config option like `lf` does to toggle wrapping.

https://github.com/gokcehan/lf/blob/master/doc.md#wrapscroll-bool-default-false

**Reply by sxyazi** [reply]
It's now a built-in feature with https://github.com/sxyazi/yazi/pull/2485 🎉
